src/generics/requests.js

```diff
@@ -81,7 +81,44 @@ var post = function (url, body, token = '', internal_access_token = false) {
 	})
 }
 
+var deleteRequest = function (url, token = '', internal_access_token = false) {
```

Comment:
@sumanvpacewisdom this is not required, user will be deleted in user service first.